uniffi_bindgen/src/bindings/gecko_js/gen_gecko_js.rs

```diff
@@ -103,13 +103,13 @@ impl<'config, 'ci> Context<'config, 'ci> {
         format!("{}_ForeignBytes", self.ci.ffi_namespace())
     }
 
-    /// Returns the `RustError` type name.
+    /// Returns the `RustCallStatus` type name.
     ///
-    /// A `RustError` is a Plain Old Data struct that holds an error code and
-    /// a message string. See the docs for `ffi_rustbuffer_type` about why this
-    /// type name must be unique for each component.
-    pub fn ffi_rusterror_type(&self) -> String {
-        format!("{}_RustError", self.ci.ffi_namespace())
+    /// A `RustCallStatus` is a Plain Old Data struct that holds an call status code and
+    /// `RustBuffer`. See the docs for `ffi_rustbuffer_type` about why this type name must be
+    /// unique for each component.
+    pub fn ffi_rustcallstatus_type(&self) -> String {
+        format!("{}_RustCallStatus", self.ci.ffi_namespace())
```

Comment:
I was really just guessing on the gecko JS code.  Is there a good way to test the JS bindings?

Reply:
There is not; TBH I think at this point we should just remove them because they're not actually used for anything in production.